### ISSUE TITLE
Symbolic hours (unicode emojis)

### ISSUE THREAD

**Issue by tinxx** [aragaer/jtt_android#22]
Hey @aragaer ,
please consider implementing the option to display unicode emojis instead of text like so:

![emoji](https://user-images.githubusercontent.com/13608545/27236374-11b71e2c-52c5-11e7-95f0-d0bfb838a157.png)
(It actually works, I added a fake translation containing the symbols (e.g. `<item>🐒</item>`).

Here is a list of emojis you could use:

```
rooster \u1F413   🐓
dog     \u1F415   🐕
pig     \u1F416   🐖
rat     \u1F400   🐀
ox      \u1F402   🐂
tiger   \u1F405   🐅
rabbit  \u1F407   🐇
dragon  \u1F409   🐉
snake   \u1F40D   🐍
horse   \u1F40E   🐏
ram     \u1F40F   🐎
monkey  \u1F412   🐒
```
---
**Update:** I sent you a patch that implements this feature.

**Comment by aragaer** [comment]
Implemented in pull request #23

**Comment by tinxx** [comment]
Nice, thanks 👍
